test/unit/job_invocation_composer_test.rb

```diff
@@ -45,29 +45,29 @@
     describe '#available_templates' do
       it 'obeys authorization' do
         composer # lazy load composer before stubbing
-        JobTemplate.expects(:authorized).with(:view_job_templates).returns(JobTemplate.scoped)
```

Comment:
In rails 3, `all` returns array, while `scoped` returns the relation. Since it seems it's the only thing that beaks the Foreman 1.10 cpmopatibilty. I suggest replacing `scoped` with `where({})` instead, not the nicest (we can leave a TODO comment to replace it once there are more reasons to deprecate 1.10